apex/transformer/parallel_state.py

```diff
@@ -84,6 +86,7 @@ def initialize_model_parallel(
     pipeline_model_parallel_size_: int = 1,
     virtual_pipeline_model_parallel_size_: Optional[int] = None,
     pipeline_model_parallel_split_rank_: Optional[int] = None,
+    use_fp8_: Optional[bool] = None,
```

Comment:
would the behavior with `use_fp8_=None` be different from that with `use_fp8_=False`?
I'm not sure about the need to make this `Optional` at glance

Reply:
If the behavior will be the same, then I'd use 
```suggestion
    use_fp8_: bool = False,
```